packages/trader/src/Modules/Trading/Helpers/digits.ts

```diff
@@ -0,0 +1,5 @@
+const digitCategoriesMap = ['even_odd', 'match_diff', 'over_under'];
```

Comment:
I would like to add `as const` to `digitCategoriesMap` and `digitTypesMap` but it leads to a bunch of problems:
1) if (`trade_type: string)` then --> string will not be assignable to type 'even_odd'| 'match_diff'|'over_under'
2) if `(trade_type: 'even_odd'| 'match_diff'|'over_under')` or any other references to array, than it wouldn't be possible to pass another trade/contract type arguments to the function (but we need that option).
3)if I delete `as const` and add only `(trade_type: string)`, it'll looks also not so good to me.
I've got an advice to make use of libraries like [ts-reset](https://github.com/total-typescript/ts-reset) in such situations, but in that case we should add it in project dependencies.

Reply:
@kate-deriv You can also add the `as const` and then change the `includes` method to `some`.
```diff
- digitCategoriesMap.includes(trade_type)
+ digitCategoriesMap.some(item => item === trade_type)
```